app/controllers/api/v1/most_items_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::MostItemsController < ApplicationController 
+
+  def index
+    render json: Item.joins(:invoice_items)
```

Comment:
@wlffann woooo business intelligence method 🎉  can we move this logic out of the controller though? where might be more appropriate?